daemon/license.go

```diff
@@ -34,7 +34,7 @@ func CheckLicense(objMgr *ibclient.ObjectManager, licenseType string) (err error
 			}
 		}
 	}
-	err = fmt.Errorf("%s License not available or User not having sufficient permissions. ", GetLicenseName(licenseType))
+	err = fmt.Errorf("%s License not available or User: %s not having sufficient permissions. ", GetLicenseName(licenseType), user)
```

Comment:
does GetLicense wapi call return error if user doesnt have sufficient permissions ? i thought its only a license check

Reply:
No it give empty list